src/app/give-help/legal-help-form/legal-help-form.component.html

```diff
@@ -0,0 +1,169 @@
+<mat-card>
+  <form (ngSubmit)="submitOffer()" #legalForm="ngForm">
+    <div class="container">
+      <div class="d-flex">
+        <button class="btn-link ms-auto" type="button" (click)="onCancelButtonClick()">
+          <mat-icon class="button-icon" fontSet="material-icons-outlined">close</mat-icon
+          ><span class="cancel-label">{{ "CANCEL" | translate }}</span>
+        </button>
+      </div>
+      <div class="row">
+        <div class="col-md-12">
+          <app-offer-title-input [(value)]="data.title"></app-offer-title-input>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12 col-md-6 col-lg-4">
+          <app-cities-search
+            [(ngModel)]="data.location"
+            name="location"
+            [placeholder]="'PLACEHOLDER_LOCATION' | translate"
+            [label]="'LOCATION' | translate"
+          ></app-cities-search>
+        </div>
+
+        <div class="col-12 col-md-6 col-lg-2">
+          <label
+            ><b>{{ "LANGUAGE" | translate }} *</b></label
+          >
+          <mat-form-field appearance="outline">
+            <mat-select
+              placeholder="{{ 'PLACEHOLDER_CHOOSE' | translate }}"
+              multiple
+              [(ngModel)]="data.language"
+              name="language"
+              required
+              #language="ngModel"
+            >
+              <mat-option *ngFor="let lang of LAW_LANGUAGES" [value]="lang.code">{{
+                lang.label | translate
+              }}</mat-option>
+            </mat-select>
+            <mat-error>
+              <app-field-error [model]="language"></app-field-error>
+            </mat-error>
+          </mat-form-field>
+        </div>
+
+        <div class="col-12 col-md-6 col-lg-3">
+          <label
+            ><b>{{ "LAW_HELP_MODES" | translate }} *</b></label
+          >
+          <mat-form-field appearance="outline">
+            <mat-select
+              placeholder="{{ 'PLACEHOLDER_CHOOSE' | translate }}"
+              multiple
+              [(ngModel)]="data.helpMode"
+              name="helpMode"
+              required
+              #helpMode="ngModel"
+            >
+              <mat-option *ngFor="let helpMode of LAW_HELP_MODES" [value]="helpMode.code">{{
```

Comment:
to chyba nie będzie potrzebne, możemy spiąć gotowe enumy z core/api i spiac bezposrednio z tlumaczneiem.
  MODE_ENUM = Object.values(JobOfferDefinitionDTO.ModeEnum);

Reply:
Zobacz w pliku `/src/app/find-help/law-search/law-search-form/law-search-form.component.html` tu już dla kategorii prawa spięte z enumami z core/api

Reply:
Dzięki panowie, tak też uczyniłem.

Reply:
Zrobiłem przy okazji rename z 'legal-help' na 'law', żeby było spójnie.